content/en/armory-enterprise/armory-admin/authn-github.md

```diff
@@ -16,7 +16,7 @@ description: >
 
 1. Login to GitHub and go to Settings > Developer Settings > OAuth Apps > New OAuth App
 2. Note the Client ID / Client Secret
-3. Homepage URL: This would be the URL of your Spinnaker service e.g. `https://spinnaker.acme.com`
+3. Homepage URL: This would be the URL of your Spinnaker Gate e.g. `https://gate.spinnaker.acme.com`
 4. Authorization callback URL: This is going to match your `--pre-established-redirect-uri` and the URL needs `login` appended to your gate endpoint e.g. `https://gate.spinnaker.acme.com/login`  or `https://spinnaker.acme.com/gate/login`
```

Comment:
@shlomodaari I noticed you removed `preEstablishedRedirectUri` from the YAML, so do we need to remove it here as well?

```suggestion
4. Authorization callback URL: The URL needs `login` appended to your Gate endpoint; for example,  `https://gate.spinnaker.acme.com/login`  or `https://spinnaker.acme.com/gate/login`.
```

Reply:
Thank you for pointing this out! Yes.